webrob/neems/manager.py

```diff
@@ -1,44 +1,44 @@
-
 import os
 from flask import session
 from webrob.neems.neem import NEEM
 
-NEEM_DIR="/neems"
+NEEM_DIR = "/neems"
```

Comment:
It would be easier to review when these auto-formatting changes were done separately. I am fine with it, but now it is hard to spot changes in this request.